### PR TITLE
=docc Move to multiple Package.swift files

### PR DESCRIPTION
**Pull request by ktoso**
As we agreed in offline discussions we'll adopt this style for the time being.

Also added the preview script

### CODE REVIEW

**Comment by ktoso** [comment]
This is addressing review comments form https://github.com/apple/swift-log/pulls?q=is%3Apr+is%3Aclosed that I missed -- moved to multiple package files

File: Package.swift [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.0
+// swift-tools-version:5.7
```
Comment (yim-lee) [inline Package.swift:1]: Keep 5.6 for now?

File: scripts/preview_docc.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+## Licensed under Apache License v2.0
+##
+## See LICENSE.txt for license information
+## See CONTRIBUTORS.md for the list of Swift Logging API project authors
+##
+## SPDX-License-Identifier: Apache-2.0
+##
+##===----------------------------------------------------------------------===##
+
+xcrun swift package --disable-sandbox preview-documentation --target Logging
```
Comment (yim-lee) [inline scripts/preview_docc.sh:16]: Can remove `xcrun`.

Comment (ktoso) [reply scripts/preview_docc.sh:16]: Follow up in https://github.com/apple/swift-log/pull/231

File: Package.swift (hunk 1 above)
Comment (ktoso) [reply Package.swift:1]: You're right thanks https://github.com/apple/swift-log/pull/231

**Pull request merged**